src/gui/dialogs/export_project.ui

```diff
@@ -81,25 +81,57 @@
          </widget>
         </item>
         <item>
+         <widget class="QWidget" name="depthWidget" native="true">
+          <property name="minimumSize">
+           <size>
+            <width>0</width>
+            <height>20</height>
+           </size>
+          </property>
+          <layout class="QVBoxLayout">
+           <property name="margin">
+            <number>0</number>
+           </property>
+           <item>
+            <widget class="QLabel" name="labelBitDepth">
+             <property name="text">
+              <string>Depth:</string>
+             </property>
+            </widget>
+           </item>
+           <item>
+            <widget class="QComboBox" name="depthCB">
+             <item>
+              <property name="text">
+               <string>16 Bit Integer</string>
+              </property>
+             </item>
+             <item>
+              <property name="text">
+               <string>24 Bit Float</string>
```

Comment:
Integer is a common English word. Also, Both Bitwig and FL Studio use the `16 Bit Integer, 24 Bit Integer, 32 Bit Float` naming. 

Whoever's working with digital audio will need to learn the difference between integer and floats. PCM is not helpful at all since they all use PCM, and as such you can omit that from the labels. Also, if you search for bit depth, the first thing that'll pop out is a very useful wiki article that explains it all.